examples/esp-chat-react/js/components/ThreadListItem.react.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+var React = require('react');
+var cx = require('react/lib/cx');
+
+var ThreadListItem = React.createClass({
+
+    _onClick: function() {
+        var thread = this.props.model;
+        this.props.router.publishEvent(this.props.modelId, "threadSelected", { threadId: thread.id, threadName: this.props.model.name });
+    },
+
+    render: function () {
+        var thread = this.props.model;
+        return (
+            <li
```

Comment:
I still think that forcing components to a specific layout isn't the way to go.  Have the components simple a components on their own and let the consumer choose how they're displayed